pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -34,14 +35,16 @@ parts = rf.select(
 )
 
 print("CRS", crs.value.crsProj4)
-parts.show(5, False)
+```
 
+```python,  show_parts, results='raw'
+parts.show(5, truncate=False)
```

Comment:
for whatever reason this one renders with line wraps in the cells for me. so it does look nice.